src/main/scala/nl/knaw/dans/easy/fedora2vault/Command.scala

```diff
@@ -39,8 +42,13 @@ object Command extends App with DebugEnhancedLogging {
 
   private def runSubcommand(app: EasyFedora2vaultApp): Try[FeedBackMessage] = {
     val outputDir = commandLine.outputDir()
-    commandLine.datasetId
-      .map(app.simpleTransform(_, outputDir)) // TODO curry to get rid of _, after/when merging with PR #2
-      .getOrElse(app.simpleTransForms(commandLine.inputFile(), outputDir))
+    implicit val logFile: File = commandLine.logFile.map(identity)
+      .getOrElse(File(s"easy-fedora2vault-${ DateTime.now.toString("yyyy-MM-dd_hh-mm-ss") }.csv"))
```

Comment:
Work sheet creates the intended file name, the real thing not
![image](https://user-images.githubusercontent.com/10553630/81289417-b1118200-9066-11ea-99c6-03bbcb81a49e.png)